### PR TITLE
Added FEVEROUS dataset

### PR DESCRIPTION
**Pull request by christos-c**
*Description of changes:*
Added the FEVEROUS dataset

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: e5e1957d739ec1eb9f82a15fae51f6a00f65cd82
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=adee948e-74c2-4a6a-824a-702c1b461c81%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 30c883f21e60035bf009131a6a08f33b66de02d1
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=4fabafa9-d7c2-435f-92f5-e4c26fbe524d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/feverous.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ManagedBy: '[FEVER team](https://fever.ai)'
+UpdateFrequency: Not updated
+Tags:
+  - natural language processing
+  - machine learning
+  - fact verification
+License: '[CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)'
+Resources:
+  - Description: Project data files in a public bucket
+    ARN: arn:aws:s3:::fever.public/feverous/
```
Comment (jflasher) [inline datasets/feverous.yaml:17]: It does not look like this S3 bucket is publicly accessible? You can see an example 'open data' bucket policy at https://gist.github.com/jedsundwall/a5d07ae3971fb172cf923d8de4625536.

File: datasets/feverous.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    Region: eu-west-1
+    Type: S3 Bucket
+DataAtWork:
+  Tools & Applications:
+    - Title: FEVEROUS Annotation Tool & Baseline
+      URL: https://github.com/Raldir/FEVEROUS
+      AuthorName: Rami Aly
+  Publications:
```
Comment (jflasher) [inline datasets/feverous.yaml:25]: Could you please list out the publications individually instead of having one placeholder for them?

**Review by jflasher — CHANGES_REQUESTED** [review]
Thanks for submitting! Left a couple of comments.

File: datasets/feverous.yaml (hunk 1 above)
Comment (christos-c) [reply datasets/feverous.yaml:17]: I have made the individual objects public e.g. https://s3-eu-west-1.amazonaws.com/fever.public/feverous/train.jsonl

As this is an Amazon AWS account, we have been asked not to make S3 buckets themselves publicly available. Is this a deal-breaker for the registry? 

If it helps, we can have an internal chat on Chime/Slack - I'm chrchrs@

File: datasets/feverous.yaml (hunk 2 above)
Comment (christos-c) [reply datasets/feverous.yaml:25]: We are waiting for the peer review results on our dataset publication - I was planning to update with a link once the paper gets accepted.

**Pull request closed without merging**